pages/curriculum.tsx

```diff
@@ -130,13 +132,17 @@ export const Curriculum: React.FC<Props> = ({ lessons, alerts }) => {
   }, [])
   useEffect(() => {
     if (data && data.session) {
+      if (data.session.user && data.session.user.isConnectedToDiscord) {
+        router.push('/discord/connect')
```

Comment:
This also doesn't seem to be taking into account the user pressing the `no thank` button as it keeps send me right back to the connect page

Reply:
Another error is that it should be !data.session.user.isConnectedToDiscord

the no thanks button has been updated in another PR to (unceremoniously) log out the user.

Reply:
seems harsh